module/enrichers.mjs

```diff
@@ -870,13 +921,21 @@ async function rollAction(event) {
  * @returns {Promise<void>}
  */
 async function rollDamage(event) {
-  const target = event.target.closest(".roll-link");
-  const { formula, damageType } = target.dataset;
+  const target = event.target.closest(".roll-link-group");
+  let { formulas, damageTypes, rollType } = target.dataset;
+  formulas = formulas.split("&");
+  damageTypes = damageTypes.split("&");
 
   const rollConfig = {
     event,
     hookNames: ["damage"],
-    rolls: [{ parts: [formula], options: { type: damageType } }]
+    rolls: formulas.map((formula, idx) => {
+      const types = damageTypes[idx]?.split("|") ?? [];
```

Comment:
```suggestion
      const types = damageTypes[idx]?.split(/[|/]/g) ?? [];
```

Reply:
I decided to normalize to `|` in the config data, so you can use either `/` or `|` but the system doesn't have to worry about it later.